paasta_tools/paastakube/api_client.py

```diff
@@ -0,0 +1,684 @@
+"""
+    Kubernetes
+
+    No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+
+    OpenAPI spec version: v1.10.1
+
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+import json
+import mimetypes
+import os
+import re
+import tempfile
+from datetime import date
+from datetime import datetime
+from multiprocessing.pool import ThreadPool
+
+from kubernetes.client import models
+from kubernetes.client.configuration import Configuration
+from kubernetes.client.rest import ApiException
+from kubernetes.client.rest import RESTClientObject
+from six import integer_types
+from six import iteritems
+from six import text_type
+from six.moves.urllib.parse import quote
+# python 2 and python 3 compatibility library
+
+
+def set_disrupted_pods(self, disrupted_pods):
+    self._disrupted_pods = disrupted_pods
+
+
+models.V1beta1PodDisruptionBudgetStatus.disrupted_pods = property(
+    fget=models.V1beta1PodDisruptionBudgetStatus.disrupted_pods,
```

Comment:
Also, since `models.V1beta1PodDisruptionBudgetStatus.disrupted_pods` is a `property`, it can't be used as a getter here (that's why that "TypeError: 'property' object is not callable" error).  You can either introduce `get_disrupted_pods` function and use it here as a getter, or try to extract something callable fro the property.